### ISSUE TITLE
Failed to resolve: com.idevicesinc:sweetbluerx:4.0.0.5-SNAPSHOT

### ISSUE THREAD

**Issue by thaiphandinh** [HubbellCorp/SweetBlue#16]
![image](https://user-images.githubusercontent.com/79837212/185621625-3b727edc-23be-48a8-8aa3-2d590f95e159.png)

It works with `com.idevicesinc:sweetblue:4.0.0 `but not `com.idevicesinc:sweetbluerx:4.0.0.5-SNAPSHOT`.
Also `com.idevicesinc:sweetbluerx:4.0.0` seems to work but it contains nothing.

**Comment by ryanhubbell** [comment]
SNAPSHOT versions are not release versions. They are test versions, and aren't meant for production use.

Are you adding `@aar` at the end? so `com.idevicesinc:sweetbluerx:4.0.0@aar`

**Comment by thaiphandinh** [comment]
Adding `@aar` files works in my current use. 

I've just tried to setup another one following [this](https://github.com/HubbellCorp/SweetBlue/wiki/Getting-Started#getting-started) pulling the library from maven server but failed. 

So, if we have to add `@aar` files to make it works, then https://github.com/HubbellCorp/SweetBlue/wiki/Getting-Started#getting-started and https://github.com/HubbellCorp/SweetBlue/wiki/Getting-Started-Rx should be updated then?

**Comment by ryanhubbell** [comment]
No, it's a current bug that has a workaround of adding `@aar`. See https://github.com/HubbellCorp/SweetBlue/issues/9

**Comment by thaiphandinh** [comment]
> No, it's a current bug that has a workaround of adding `@aar`. See #9

Thank you for pointing this out. I believe this should be mentioned on getting started pages.

**Comment by jamesagada** [comment]
Still doesn't work.
